### ISSUE TITLE
AMBER Installation

### ISSUE THREAD

**Issue by alext1202** [ubccr/software-layer#208]
Hi, I would like AMBER (a MD software) to be installed. Here is the link to the website: https://ambermd.org/ ...Thank you!

**Comment by tonykew** [comment]
A library problem was identified with the build - fixing now...

**Comment by tonykew** [comment]
Updated test builds complete - more info to follow...

**Comment by tonykew** [comment]
amber/22.0-AmberTools-23.0  and  amber/22.0-AmberTools-23.0-CUDA-11.8.0  have been published to the ccrsoft/2023.01 software release


```
vortex1$ module spider amber
----------------------------------------------------------------------------
  amber:
----------------------------------------------------------------------------
    Description:
      Amber (originally Assisted Model Building with Energy Refinement) is
      software for performing molecular dynamics and structure prediction.

     Versions:
        amber/22.0-AmberTools-23.0-CUDA-11.8.0
        amber/22.0-AmberTools-23.0
[...]
vortex1$ 
```

non CUDA (non GPU) version:

```
vortex1$ module spider amber/22.0-AmberTools-23.0
----------------------------------------------------------------------------
  amber: amber/22.0-AmberTools-23.0
----------------------------------------------------------------------------
    Description:
      Amber (originally Assisted Model Building with Energy Refinement) is
      software for performing molecular dynamics and structure prediction.

    Properties:
      Chemistry libraries/apps

    You will need to load all module(s) on any one of the lines below before the "amber/22.0-AmberTools-23.0" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
vortex1$ module load gcc/11.2.0 openmpi/4.1.1 amber/22.0-AmberTools-23.0

For academic use only, following the University of California license
restrictions described here:
  https://ambermd.org/GetAmber.php

vortex1$ 
vortex1$ ls ${EBROOTAMBER}/bin
addles                      makeCSA_RST.na                 prepgen
add_pdb                     makeDIP_RST.dna                process_mdout.perl
AddToBox                    makeDIP_RST.protein            process_minout.perl
add_xray                    makeDIST_RST                   PropPDB
am1bcc                      makeRIGID_RST                  ProScrs.py
amb2chm_par.py              MCPB.py                        pyresp_gen.py
amb2chm_psf_crd.py          mdgx                           py_resp.py
amb2gro_top_gro.py          mdgx.MPI                       quick
ambmask                     mdgx.OMP                       quick.MPI
ambpdb                      mdout2pymbar.pl                reduce
antechamber                 mdout_analyzer.py              residuegen
ante-MMPBSA.py              memembed                       resp
atomtype                    metalpdb2mol2.py               respgen
bar_pbsa.py                 metatwist                      rism1d
bondtype                    mm_pbsa_nabnmode               rism3d.orave
CartHess2FC.py              mm_pbsa.pl                     rism3d.snglpnt
car_to_files.py             MMPBSA.py                      rism3d.snglpnt.MPI
ceinutil.py                 mmpbsa_py_energy               rism3d.thermo
cestats                     MMPBSA.py.MPI                  sander
charmmlipid2amber.py        mmpbsa_py_nabnmode             sander.LES
ChBox                       mm_pbsa_statistics.pl          sander.LES.MPI
cpeinutil.py                mol2rtf.py                     sander.MPI
cphstats                    ndfes                          sander.OMP
cpinutil.py                 ndfes-AvgFESs.py               saxs_md
cpptraj                     ndfes-CheckEquil.py            saxs_md.OMP
cpptraj.MPI                 ndfes-CombineMetafiles.py      saxs_rism
cpptraj.OMP                 ndfes-FTSM-PrepareAnalysis.py  saxs_rism.OMP
draw_membrane2              ndfes-FTSM-PrepareGuess.py     senergy
edgembar                    ndfes-FTSM-PrepareSims.py      sgldinfo.sh
edgembar-amber2dats.py      ndfes.OMP                      sgldwt.sh
edgembar.OMP                ndfes-PrepareAmberData.py      simplepbsa
edgembar-WriteGraphHtml.py  ndfes-PrintFES.py              simplepbsa.MPI
elsize                      ndfes-PrintStringFES.py        softcore_setup.py
espgen                      nef_to_RST                     sqm
espgen.py                   nfe-umbrella-slice             sqm.MPI
FEW.pl                      nmode                          sviol
finddgref.py                OptC4.py                       sviol2
fitpkaeo.py                 packmol                        teLeap
fixremdcouts.py             packmol-memgen                 test-api
gbnsr6                      paramfit                       test-api.MPI
gem.pmemd                   paramfit.OMP                   tinker_to_amber
gem.pmemd.MPI               parmcal                        tleap
genremdinputs.py            parmchk2                       ucpp
gwh                         parmed                         UnitCell
hcp_getpdb                  pbsa                           wrapped_progs
IPMach.py                   pdb4amber                      xaLeap
makeANG_RST                 PdbSearcher.py                 xleap
makeCHIR_RST                pmemd                          XrayPrep
make_crd_hg                 pmemd.MPI
vortex1$ 
```
running "sander" as per the tutorial (for example)

```
vortex1$ sander -O -i 01_Min.in -o 01_Min.out -p parm7 -c rst7 -r 01_Min.ncrst \
> -inf 01_Min.mdinfo
vortex1$ 
```


For the CUDA (GPU) version:

```
vortex1$ module load gcc/11.2.0 openmpi/4.1.1 amber/22.0-AmberTools-23.0-CUDA-11.8.0

For academic use only, following the University of California license
restrictions described here:
  https://ambermd.org/GetAmber.php

vortex1$ 
```

This version includes (adds) CUDA specific binaries:

```
vortex1$ ls ${EBROOTAMBER}/bin | grep cuda
cpptraj.cuda
mdgx.cuda
pbsa.cuda
pmemd.cuda
pmemd.cuda_DPFP
pmemd.cuda_DPFP.MPI
pmemd.cuda.MPI
pmemd.cuda_SPFP
pmemd.cuda_SPFP.MPI
quick.cuda
quick.cuda.MPI
sander.quick.cuda
sander.quick.cuda.MPI
test-api.cuda
test-api.cuda.MPI
vortex1$ 
```
